vhdbuilder/packer/windows-image.env

```diff
@@ -10,7 +10,19 @@ WINDOWS_2019_BASE_IMAGE_VERSION=17763.3165.220706
 WINDOWS_2022_BASE_IMAGE_SKU=2022-Datacenter-Core-smalldisk
 WINDOWS_2022_BASE_IMAGE_VERSION=20348.825.220704
 
+# Please refer to https://vincentlauzon.com/2018/01/10/finding-a-vm-image-reference-publisher-sku/
+# , https://docs.microsoft.com/en-us/azure/virtual-machines/windows/ps-template for ARM templates.
+# Please use this PowershellScript to check for available skus that azure-arm can use:
+#   Get-AzureRmVMImageSku -Location eastus -PublisherName MicrosoftWindowsServer  -Offer WindowsServer 
```

Comment:
Can we use `azure cli` as the example as we did for gen1?

Reply:
Changed to the `azure cli` example. Would you take a look https://github.com/Azure/AgentBaker/pull/2036/commits/7dfa348264f6453af6a2be809c1b517b66b94de8?